tests/core/evaluation/test_stats.py

```diff
@@ -0,0 +1,255 @@
+import os
+import math
+import numpy as np
+from rasa.core.evaluation.stats import (
+    load_extracted_markers_json_file,
+    compute_multi_tracker_stats,
+    compute_summary_stats,
+    compute_single_tracker_stats,
+    write_stats,
+)
+
+
+def test_compute_summary_stats():
+    """Tests compute summary stats."""
+    a = [20, 30, 32]
+    stats_a = compute_summary_stats(a)
+    assert stats_a["n"] == 3
+    assert math.isclose(stats_a["mean"], 27.33, abs_tol=0.01)
+    assert stats_a["median"] == 30
+    assert stats_a["min"] == 20
+    assert stats_a["max"] == 32
+
+    b = [15]
+    stats_b = compute_summary_stats(b)
+    assert stats_b["n"] == 1
+    assert stats_b["mean"] == 15
+    assert stats_b["median"] == 15
+    assert stats_b["min"] == 15
+    assert stats_b["max"] == 15
+
+
+def test_compute_summary_stats_empty():
+    """Tests compute summary stats on an empty array."""
+    stats = compute_summary_stats([])
+    assert stats == {
+        "n": 0,
+        "mean": np.nan,
+        "median": np.nan,
+        "min": np.nan,
+        "max": np.nan,
+    }
+
+
+def test_compute_single_tracker_stats(extracted_markers_json):
```

Comment:
There's a convention that we should type test parameters too. You'll probably need to add `from typing import Text` (I think that's correct) to pull the type in as well
```suggestion
def test_compute_single_tracker_stats(extracted_markers_json: Text):
```